ipatests/test_integration/test_adtrust_install.py

```diff
@@ -216,3 +220,88 @@ def test_schema_compat_attribute(self):
         value = (r'ipaexternalmember=%deref_r('
                  '"member","ipaexternalmember")')
         assert value in entry_list
+
+
+class TestIpaNisManage(IntegrationTest):
+    """
+    Tests for ipa-nis-manage CLI tool
+    """
+    @classmethod
+    def install(cls, mh):
+        tasks.install_master(cls.master)
+
+    def test_ipa_nis_manage_enable(self):
+        """
+        This testcase checks if ipa-nis-manage enable
+        command enables plugin on an IPA master
+        """
+        serverid = realm_to_serverid(self.master.domain.realm)
+        dirsrv_service = "dirsrv@%s.service" % serverid
```

Comment:
This can be moved to `install()` method.
```
        self.serverid = realm_to_serverid(self.master.domain.realm)
        self.dirsrv_service = "dirsrv@%s.service" % self.serverid
```
 By doing so you will not have to define it in each test. You can simply use `self.master.run_command(["systemctl", "restart", self.dirsrv_service])` further in testcases.